node/utils/i18n.ts

```diff
@@ -32,8 +40,39 @@ export const addContextToTranslatableString = (message: Message, ctx: Context) =
     from: originalFrom
   } = parseTranslatableStringV2(message.content)
 
-  const context = originalContext || message.context
+  const context = (originalContext || message.context).toString()
   const from = originalFrom || message.from || locale
 
-  return formatTranslatableStringV2({content, context, from})
+  return formatTranslatableStringV2({ content, context, from })
+}
+
+export const translateToCurrentLanguage = (message: MessageWithContext, ctx: Context) => {
```

Comment:
Why it can receive only a message with context?

Reply:
Because I want for now, but it is not needed. I see no harm. If a future dev wants, it may adapt.